plugins/inmemoryscanner/src/lib/Scanner.cpp

```diff
@@ -215,14 +201,16 @@ namespace InMemoryScanner
     {
         for (const auto& rule : results)
         {
-            std::string matchesAsString(rule.ruleNamespace + " : " + rule.ruleName + " in " + processName + " (" +
-                                        std::to_string(pid) + ") at " + intToHex(base) + " matches (");
             for (const auto& match : rule.matches)
             {
-                matchesAsString += match.matchName + ", ";
+                inMemResultsLogger->info("Rule matches",
```

Comment:
this will create a new line for every match from a rule in contrast to all matches being concatenated. Is this intentional?

Reply:
Yes